src/commands/aksAccount/aksAccount.ts

```diff
@@ -106,3 +101,59 @@ export async function selectSubscriptions(): Promise<void> {
 
     await setFilteredSubscriptions(newFilteredSubscriptions);
 }
+
+export async function selectSubscription(): Promise<string | undefined> {
+    const sessionProvider = await getReadySessionProvider();
```

Comment:
can we make a util function for common things between selectSubscriptions() and new method selectSubscription() ?

Reply:
Yes, good idea. Updated